contents/python/introduction-to-python.md

```diff
@@ -116,4 +108,11 @@ Learning how to avoid such pitfalls is one huge step towards being a productive
 
 * [Top 10 Common Mistakes of Python Programmers](https://www.toptal.com/python/top-10-mistakes-that-python-programmers-make)
 * [Common Python Gotchas](https://sopython.com/wiki/Common_Gotchas_In_Python)
+
+## A Note on Python 2
+When reading about Python online, you may encounter references to *Python 2*. Python 2 is an older, **discontinued** version of the Python language.
+The current major version of Python, Python 3, is backwards incompatible with Python 2. Read this [article](https://www.geeksforgeeks.org/important-differences-between-python-2-x-and-python-3-x-with-examples/) to learn about some of the differences between Python 2 and 3.
```

Comment:
```suggestion
The current major version of Python, Python 3, is backwards incompatible with Python 2. You may wish to read this [article](https://www.geeksforgeeks.org/important-differences-between-python-2-x-and-python-3-x-with-examples/) to learn more about some of the differences between Python 2 and 3.
```
Just a small nit, I think "Read this" sounds too much like an instruction rather than a suggestion. Feel free to change it to something other than "You may wish to read", that's the best phrasing I can think of right now.

Reply:
Thanks, yes "You may wish to read" sounds better to me.
However, for your second change: I think the sentence should still be "to learn about some of the differences", rather than "to learn **more** about some of the difference".
Since we are not covering anything about the differences in this article, we should not use the phrase "to learn more about" ?